pyquil/tests/test_gates.py

```diff
@@ -1,5 +1,6 @@
 import pytest
-from pyquil.gates import *
```

Comment:
is `import *` considered bad style? Even in tests?

Reply:
This change was to appease mypy; otherwise, it gets confused and doesn't know where to find the definitions of the gates and complains that they are "undefined".